node/cn/peer.go

```diff
@@ -732,8 +733,8 @@ func (p *basePeer) UpdateRWImplementationVersion() {
 }
 
 // RegisterConsensusMsgCode is not supported by this peer.
-func (p *basePeer) RegisterConsensusMsgCode(msgCode uint64) {
-	logger.Error("RegisterConsensusMsgCode is not supported by this peer.")
+func (p *basePeer) RegisterConsensusMsgCode(msgCode uint64) error {
+	return fmt.Errorf("%v peerID: %v ", errNotSupportedByBasePeer, p.GetID())
```

Comment:
Now we can use Wrapping errors. :)

https://blog.golang.org/go1.13-errors?fbclid=IwAR10-LqfBjvtWqtqmjcAi-SaRGs4LrvcLhaKDBQi5ir_taYYqKL_et31i-U

```suggestion
	return fmt.Errorf("%w peerID: %v ", errNotSupportedByBasePeer, p.GetID())
```